### ISSUE TITLE
TypeError: Cannot read property 'replace' of undefined

### ISSUE THREAD

**Issue by miguelmota** [jscs-dev/node-jscs#2243]
jscs doesn't work with [es function bind](https://github.com/zenparsing/es-function-bind) syntax.

`index.js`

```javascript
foo::bar()
```

```bash
$ jscs index.js

TypeError: Cannot read property 'replace' of undefined
    at renderLine (/Users/mota/.nvm/versions/node/v6.0.0/lib/node_modules/jscs/lib/errors.js:266:16)
    at Object.Errors.explainError (/Users/mota/.nvm/versions/node/v6.0.0/lib/node_modules/jscs/lib/errors.js:197:13)
    at /Users/mota/.nvm/versions/node/v6.0.0/lib/node_modules/jscs/lib/reporters/console.js:16:36
    at Array.forEach (native)
    at /Users/mota/.nvm/versions/node/v6.0.0/lib/node_modules/jscs/lib/reporters/console.js:14:35
    at Array.forEach (native)
    at Object.module.exports [as writer] (/Users/mota/.nvm/versions/node/v6.0.0/lib/node_modules/jscs/lib/reporters/cons
ole.js:9:22)
    at /Users/mota/.nvm/versions/node/v6.0.0/lib/node_modules/jscs/lib/cli.js:149:18
    at Array.<anonymous> (/Users/mota/.nvm/versions/node/v6.0.0/lib/node_modules/jscs/node_modules/vow/lib/vow.js:712:56
)
    at Immediate.callFns [as _onImmediate] (/Users/mota/.nvm/versions/node/v6.0.0/lib/node_modules/jscs/node_modules/vow
/lib/vow.js:23:35)
```

**Comment by markelog** [comment]
Need to add support for this in cst first - https://github.com/jscs-dev/node-jscs/issues/2243

**Comment by markelog** [comment]
Fixed via a215e672ad76270881bfc7252a954762119c48a3
